src/main/java/org/broadinstitute/hellbender/utils/reference/ReferenceUtils.java

```diff
@@ -107,4 +116,38 @@ public static byte[] getRefBaseAtPosition(final ReferenceSequenceFile reference,
     public static byte[] getRefBasesAtPosition(final ReferenceSequenceFile reference, final String contig, final int start, final int length) {
         return reference.getSubsequenceAt(contig, start, start+length-1).getBases();
     }
+
+    /**
+     * Given a reference data source and a sequence interval, calculates the MD5 for the given sequence.
+     *
+     * Note: does not close the ReferenceDataSource it's passed.
```

Comment:
Can you add a to this comment a line stating that this md5 code mimics the MD5 calculation code in `{@link picard.sam.CreateSequenceDictionary}` in that it allws IUPAC bases but uppercases all the bases.